manifests/init.pp

```diff
@@ -57,6 +57,11 @@
 #   Enable IP masquerading for bridge's IP range.
 #   The default is true.
 #
+# [*icc*]
+#   Enable Docker's unrestricted inter-container and Docker daemon host communication.
+#   (Requires iptables=true to disable)
+#   Default is true.
```

Comment:
This says the default is true, but in params it's set to undefined. I think it's worth being clear, explicit true is different to undefined and implicitly true because of defaults in Docker for instance.